CRM/Contact/BAO/Query.php

```diff
@@ -1828,6 +1830,12 @@ public function whereClauseSingle(&$values, $isForcePrimaryOnly = FALSE) {
       $this->buildRelativeDateQuery($values);
       return;
     }
+
+    // Handle situations where we have already gone through function convertTextStringsToUseLikeOperator.
```

Comment:
@eileenmcnaughton this moves the handling that was added for the sortName field but tries to more generically handle it such that if we are in an array then set the $op to be LIKE and set the $value part of the key back to be the string rather than an arrray

Reply:
Actually this is still not working for me - let's pull it out & do it as a separate PR

Reply:
ok have pulled this change out